TM1py/Services/CellService.py

```diff
@@ -1393,10 +1393,14 @@ def extract_cellset_composition(self, cellset_id: str, sandbox_name: str = None,
         cube = response_json["Cube"]["Name"]
 
         rows, titles, columns = [], [], []
-        if response_json["Axes"][0]["Hierarchies"]:
-            columns = [hierarchy["UniqueName"] for hierarchy in response_json["Axes"][0]["Hierarchies"]]
-        if response_json["Axes"][1]["Hierarchies"]:
-            rows = [hierarchy["UniqueName"] for hierarchy in response_json["Axes"][1]["Hierarchies"]]
+        if len(response_json["Axes"]) == 1:
+            if response_json["Axes"][0]["Hierarchies"]:
+                rows = [hierarchy["UniqueName"] for hierarchy in response_json["Axes"][0]["Hierarchies"]]
```

Comment:
If only one axis is defined, it must be the columns, don't you think?

Reply:
Fixed it just now.